utils/spacewalk-common-channels.ini

```diff
@@ -901,6 +901,46 @@ gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/openSUSE_Leap_15-Uyuni-Client-Tools/openSUSE_Leap_15.0/
 
+[opensuse_leap15_1-aarch64]
+checksum = sha256
+archs    = aarch64
+name     = openSUSE Leap 15.1 (%(arch)s)
+gpgkey_url =  http://download.opensuse.org/ports/aarch64/distribution/leap/15.1/repo/oss/repodata/repomd.xml.key
+gpgkey_id = 3DBDC284
+gpgkey_fingerprint = 22C0 7BA5 3417 8CD0 2EFE  22AA B88B 2FD4 3DBD C284
+repo_url = http://download.opensuse.org/distribution/leap/15.1/repo/oss/
+dist_map_release = 15.1
+
+[opensuse_leap15_1-updates-aarch64]
+label    = %(base_channel)s-updates
+name     = openSUSE Leap 15.1 Updates (%(arch)s)
+archs    = aarch64
+checksum = sha256
+base_channels = opensuse_leap15_1-%(arch)s
+repo_url = http://download.opensuse.org/ports/update/leap/15.1/
```

Comment:
But since here we have this I wonder: @mcalmer, should we add a set of repositories for the ports? Only problem, of course, is that not all of them are supported right now, so maybe it's best if we do it as in this PR.

Reply:
hmm, multiarch repo. I hope it still works.

Reply:
Maybe we get problems when you want to use this URL for other archs. When this comes and cause problems, we may need to fix spacewalk-common-channels